addons/cookielifetime.sh

```diff
@@ -68,14 +68,15 @@ You can not set a value below 30 minutes (1800 seconds).")
             then
                 msg_box "It seems like you weren't satisfied with your setting of ($COOKIE_LIFETIME) seconds. Please try again."
             else
-                if nextcloud_occ config:system:set remember_login_cookie_lifetime --value="$COOKIE_LIFETIME"
-                then
-                    msg_box "Cookie Lifetime is now successfully set to $COOKIE_LIFETIME seconds."
-                fi
                 break
             fi
         done
     ;;
     *)
     ;;
 esac
+
+# Set the value
+nextcloud_occ config:system:set remember_login_cookie_lifetime --value="$COOKIE_LIFETIME"
+nextcloud_occ config:system:set session_lifetime --value="$COOKIE_LIFETIME"
```

Comment:
After a more indepth testing I found out that you get logged out if either of those values gets reached due to inactivity. This was so long not clear because session_lifetime is by default set to 24h which makes testing this a bit hard. But it gets obvious if you test with 5min or 10min intervals.